.github/workflows/test.yaml

```diff
@@ -34,3 +34,17 @@ jobs:
         run: yarn test
         env:
           CI: true
+
+  inclusive-naming-check:
+  runs-on: ubuntu-latest
```

Comment:
Is this nested properly? I think it should be indented one more level (together with lines below).